generate-certs.sh

```diff
@@ -68,7 +68,8 @@ command -v certstrap > /dev/null 2>&1 || {
     sleep 1
   done
   printf "\n"
-  go get github.com/square/certstrap
+  docker run --rm -v "${HOME}/bin":/out:rw golang:1.7 /usr/bin/env GOBIN=/out go get github.com/square/certstrap
+  sudo chown "$(id -un):$(id -gn)" /home/vagrant/bin/certstrap
```

Comment:
The sudo assumes HOME=/home/vagrant, better use HOME as well.
That would work for me outside vagrant too.